internal/pkg/mapper/orchestrator/remote_load_test.go

```diff
@@ -89,15 +89,17 @@ func TestMapAfterRemoteLoad(t *testing.T) {
 	config := &model.Config{ConfigKey: configKey, Content: content}
 	configState := &model.ConfigState{
 		ConfigManifest: configManifest,
-		Local:          config,
+		Remote:         config,
```

Comment:
V povodnom kode bolo `typo`, ... jedna sa o remote load, tak tu ma byt nastaveny `Remote` stav.
Povodny kod mal fallback na `Local` ak `Remote` nebolo nastavene, tak sa na to neprislo a test prechadzal.